lib/browser/_asserters.js

```diff
@@ -105,19 +105,22 @@ exports.fuzzyString = function assertFuzzyString(
   let propName;
   if (typeof extract === 'string') {
     propName = getterToPropName(extract);
-    extract = _.method(extract);
+    extract = method(extract);
   } else if (typeof extract === 'function') {
     propName = getterToPropName(extract.name);
   } else {
     throw new Error(`Invalid extract: ${extract}`);
   }
+
   const doc = `${selector} should ${shouldMatch ? '' : 'not '}have ${propName}`;
 
   return new Asserter(target => {
     return Promise.resolve(target)
-      .then(extract)
+      .then(res => {
+        return extract(res);
+      })
```

Comment:
out of curiosity - why this change?

Reply:
💅  was testing something there and didn't revert